runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -28,6 +28,8 @@
 
     private ByteBuf serializedCache = null;
 
+    // last know serialized payload size.
+    @Getter
     private int lastKnownSize = NOT_KNOWN;
```

Comment:
Add comments about this variable and also states your concern mentioned in the lasted issue.

Reply:
There is a comment on this variable.
As for the concern in that issue, I addressed it in this PR. I will reference it in PR description.